realm/realm-jni/build.gradle

```diff
@@ -248,6 +248,109 @@ task deployCore(group: 'build setup', description: 'Deploy the latest version of
     }
 }
 
+task compileCoreLibForHost(group: 'build setup', description: 'Compile the core library for host from source code') {
+    doLast {
+        if (!coreSourcePath) {
+            throw new GradleException("The Realm Core library is needed to build support for Robolectric. coreSourcePath, /usr/local/lib/librealm.a or the environment REALM_CORE_HOME is needed.")
+        }
+        exec {
+            workingDir = coreSourcePath
+            environment 'REALM_ENABLE_ENCRYPTION', 'yes'
```

Comment:
<@dalinaum> I only enabled `ENABLE_ENCRYPTION`. `ENABLE_ASSERTIONS` is already disabled.  I didn’t change any other options. So is it same options with the build for Android?

```:realm-jni:compileCoreLibForHost
Using existing configuration in src/config.mk:
    REALM_VERSION         = 0.95.7
    INSTALL_PREFIX        = /usr/local
    INSTALL_EXEC_PREFIX   = /usr/local
    INSTALL_INCLUDEDIR    = /usr/local/include
    INSTALL_BINDIR        = /usr/local/bin
    INSTALL_LIBDIR        = /usr/local/lib
    INSTALL_LIBEXECDIR    = /usr/local/libexec
    MAX_BPNODE_SIZE       = 1000
    MAX_BPNODE_SIZE_DEBUG = 1000
    ENABLE_ASSERTIONS     = no
    ENABLE_ALLOC_SET_ZERO = no
    ENABLE_ENCRYPTION     = yes
    XCODE_HOME            = /Applications/Xcode.app/Contents/Developer
    IPHONE_SDKS           = iPhoneOS:iPhoneOS9.2.sdk:armv7,armv7s,arm64 iPhoneSimulator:iPhoneSimulator9.2.sdk:i386,x86_64
    IPHONE_SDKS_AVAIL     = yes
    WATCHOS_SDKS          = WatchOS:WatchOS2.1.sdk:armv7k WatchSimulator:WatchSimulator2.1.sdk:i386
    WATCHOS_SDKS_AVAIL    = yes
    TVOS_SDKS             = AppleTVOS:AppleTVOS9.1.sdk:arm64 AppleTVSimulator:AppleTVSimulator9.1.sdk:x86_64
    TVOS_SDKS_AVAIL       = yes
    ANDROID_NDK_HOME      = /usr/local/Cellar/android-ndk/r10e
```